docs/decisions/0002-installation-method.md

```diff
@@ -0,0 +1,82 @@
+# How to use template-infra with template applications
+
+* Status: proposed
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+
+Technical Story: User can follow instructions on how to use template-infra with either of the template applications [#2](https://github.com/navapbc/platform/issues/2)
+
+## Context and Problem Statement
+
+We previously decided to [separate infra and application templates into separate repos](./0001-separate-infra-and-application-templates.md). We need to decide how a user of the platform can use the infra template with one of the application templates or with their own custom application.
+
+## Considered Options
+
+1. User creates a repo from the template-infra repo, and replaces the `app` folder with one copied from an application template or their own.
+2. User copies relevant files from the template-infra repo and from the application template repo.
+
+## Decision Outcome
+
+TBD
+
+## Pros and Cons of the Options
+
+### Create project repo from template-infra repo
+
+Pros
+
+* Fewer things to copy over (github actions, pull request template)
+
+Cons
+
+* Doesn't work with existing repo (e.g. if a user already has an application and wants to add template infra to the repo)
+* May still need to have a setup script or instructions to remove/clean up files that aren't needed (e.g. the example application in template-infra is likely not needed)
```

Comment:
Maybe unrelated to this proposal but why does the `template-infra` repo need the `app/` directory?

Reply:
Great question. I was talking to @rocketnova about this.

* We considered keeping some simple `app` (arguably one even simpler than what's in `template-application-nextjs` for the purposes of testing out the template-infra)
* We considered not having `app`, and using an out of the box docker image like [hello-world](https://github.com/crccheck/docker-hello-world)
* We considered including template-application-flask and template-application-next as git-submodules for testing purposes, so that we can ensure that template-infra works with the various application templates we have

As I was thinking about this yesterday I'm thinking that maybe we could get rid of the `app` directory but figure out a way to dynamically test out the various scenarios in the above bullets. Will have to think through and write a tech spec but it might be possible

Reply:
Ah okay, I haven't taken a close look at the infra directory. I was interpreting the infra repo as just the core pieces general to any application, to get Terraform connected and state-managed in AWS.

Reply:
We're currently intending template-infra to also include infra modules for setting up the ECS service and load balancer and database, and the application repo would have just the code in the docker container, so the application repo would have the Dockerfile and all the code that goes in it.

Reply:
Ah okay, I was missing that piece, thanks.